src/main/java/com/bean00/server/RequestParser.java

```diff
@@ -31,14 +31,18 @@ public Request parseRequest() throws IOException {
 
         String requestMethod = requestWords[0];
         String requestURL = requestWords[1];
+        // TODO: check if requestURL is valid
```

Comment:
Are you sure that is the responsibility of the `RequestParser`?

Reply:
Removed the old comments